Source/Fuse.Reactive.JavaScript/Tests/UX/OptionalExplicit.ux

```diff
@@ -0,0 +1,21 @@
+<Panel ux:Class="UX.OptionalExplicit">
+	<Router ux:Name="router" />
+	<JavaScript>
+		exports.foo = 123
+		// should not be reached below, as there is a bundled file that matches
+		// this path
+		exports.OptionalExplicit = 777 
```

Comment:
Something seem off on this test. Do you mean that exports from a previous JS can be require'd in the next one? If yes, maybe you meant to do `require("foo")` in the next bit to show that it's working. Otherwise nothing shows you can `require` the exports at all here.